### PR TITLE
Re-organize and restructure standards matching

### PR DESCRIPTION
**Pull request by jorainer**
This PR introduces some changes to the matching of standards:
- Tune pre-processing settings (including alignment) to identify more features.
- Manually evaluate matched features for each standard.

### CODE REVIEW

File: match-standards-mix01.Rmd [hunk 1]
```diff
@@ -97,193 +120,620 @@ data_SP <- filterFile(data, file = which(data$matrix_pol == "Serum_POS"))
...
 
 data_WP <- findChromPeaks(data_WP, param = cwp) 
-```
 
-### Refinement
-
-```{r refinement, message = FALSE, warning = FALSE, eval = !file.exists(paste0(RDATA_PATH, "processed_data.RData"))}
-mnp <- MergeNeighboringPeaksParam(expandRt = 2.5, expandMz = 0.001,
+## Peak refinement
+mnp <- MergeNeighboringPeaksParam(expandRt = 3.5, expandMz = 0.001,
```
Comment (jorainer) [inline match-standards-mix01.Rmd:156]: Increased this parameter to eventually merge more neighboring peaks.

File: match-standards-mix01.Rmd [hunk 2]
```diff
@@ -97,193 +120,620 @@ data_SP <- filterFile(data, file = which(data$matrix_pol == "Serum_POS"))
...
 ```
 
+
 ### Identification of features matching standards
 
 We now identify all features matching likely adducts of the standards.
 
 ```{r}
-adducts <- c("[M+H]+", "[M+2H]2+", "[M+Na]+", "[M+K]+", "[M+NH4]+",
+adducts <- c("[M]+", "[M+H]+", "[M+2H]2+", "[M+Na]+", "[M+K]+", "[M+NH4]+",
```
Comment (jorainer) [inline match-standards-mix01.Rmd:231]: Added `"[M]+"` to support also pre-charged molecules

File: match-standards-mix01.Rmd [hunk 3]
```diff
@@ -97,193 +120,620 @@ data_SP <- filterFile(data, file = which(data$matrix_pol == "Serum_POS"))
...
+- The `bw` parameter for the correspondence is larger than usual because adding
+  the standards in higher concentrations caused considerable retention time
+  shifts for some.
+- The `binSize` was also slightly increased to avoid splitting of features.
 
 ```{r peakdetection, eval = !file.exists(paste0(RDATA_PATH, "processed_data.RData"))}
+## Peak detection
 cwp <- CentWaveParam(ppm = 50,
-                     peakwidth = c(2, 20),
+                     peakwidth = c(2, 18),
```
Comment (jorainer) [inline match-standards-mix01.Rmd:146]: this did most likely not have a big impact, but having less broad rt range sometimes helps to better detect peaks.

File: match-standards-mix01.Rmd [hunk 4]
```diff
@@ -97,193 +120,620 @@ data_SP <- filterFile(data, file = which(data$matrix_pol == "Serum_POS"))
...
-mnp <- MergeNeighboringPeaksParam(expandRt = 2.5, expandMz = 0.001,
+## Peak refinement
+mnp <- MergeNeighboringPeaksParam(expandRt = 3.5, expandMz = 0.001,
                                   minProp = 3/4)
 data_WP <- refineChromPeaks(data_WP, param = mnp)
-```
 
-### Correspondence analysis
+## Alignment
+pdp1 <- PeakDensityParam(sampleGroups = data_WP$matrix_pol, bw = 3,
```
Comment (jorainer) [inline match-standards-mix01.Rmd:161]: there are rt shifts in the data, specifically due to the higher concentration. The increased `bw` should help to still group such shifted peaks into the same feature.

File: match-standards-mix01.Rmd [hunk 5]
```diff
@@ -97,193 +120,620 @@ data_SP <- filterFile(data, file = which(data$matrix_pol == "Serum_POS"))
...
+                        minFraction = 0.7, binSize = 0.015)
+data_WP <- groupChromPeaks(data_WP, param = pdp1)
+pgp <- PeakGroupsParam(minFraction = 0.8, extraPeaks = 100, span = 0.8)
+data_WP <- adjustRtime(data_WP, param = pgp)
 
-Next we perform a correspondence analysis to group chromatographic peaks across
-samples.
+## Correspondence analysis
+pdp2 <- PeakDensityParam(sampleGroups = data_WP$mode, bw = 3,
+                        minFraction = 0.3, binSize = 0.015)
```
Comment (jorainer) [inline match-standards-mix01.Rmd:169]: Lowering the `minFraction` helps to define more features. (unexpectedly) for some standards no peak was identified in the low concentration.

**Review by andreavicini — APPROVED** [review]
Thanks @jorainer! Looks like you improved things a lot (and I made confusion with a few things, sorry). I will merge so that I can run code, look at the new results and proceed with the remaining standards

**Pull request merged**